llvm/test/Transforms/LoopVectorize/X86/cost-model.ll

```diff
@@ -329,83 +329,24 @@ for.end:
 define void @multi_exit(ptr %dst, ptr %src.1, ptr %src.2, i64 %A, i64 %B) #0 {
 ; CHECK-LABEL: @multi_exit(
 ; CHECK-NEXT:  entry:
-; CHECK-NEXT:    [[UMAX6:%.*]] = call i64 @llvm.umax.i64(i64 [[B:%.*]], i64 1)
```

Comment:
I think it's worth rewriting the test to so that at least one load is not loop invariant, because the test seems to care about being vectorised with multiple exits.